packages/flock/flock.1.0.0/descr

```diff
@@ -0,0 +1 @@
+Ctypes bindings to flock for OCaml
```

Comment:
woops, scratch that, it's `flock(2)` :-) I don't know why, I read it `flock` instead of `f-lock`. Sorry about the noise :-)